### PR TITLE
Add plotting to dex-arbitrage of candle data and multi-pairs

### PR DESCRIPTION
**Pull request by pmm09c**

### CODE REVIEW

File: src/dextrader/analysis/arbitrage.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    merged_data = merged_data.assign(profitable_trades_min=
+                                     lambda x: x['arbitrage_min'] > threshold)
+
+    # Get time in int64
+    merged_data['datetime64'] = merged_data['Datetime'].astype('datetime64[ns]')
+
+    arbitrage_results = dict()
+    arbitrage_results['info'] = (pair, mkt1, mkt2)
+    arbitrage_results['dataframe'] = merged_data
+    arbitrage_results['profitable_trades_min'] = np.sum(merged_data['profitable_trades_close'].to_numpy().astype(int))
```
Comment (sububer) [inline src/dextrader/analysis/arbitrage.py]: Lines 64-65, both sum `profitable_trades_close`. Is this one supposed to be `profitable_trades_min` ?

**Review by sububer — DISMISSED** [review]
Very cool. I ran it, and see the Spread Summary, candle data, and volume plots, as well as the ability to switch between exhanges and close/worst case.
See one comment in arbitrage where you're summing `profitable_trades_close`.

File: src/dextrader/analysis/arbitrage.py (hunk 1 above)
Comment (pmm09c) [reply src/dextrader/analysis/arbitrage.py]: nice catch

**Review by sububer — APPROVED** [review]
Re-ran with changes. Works as expected... all plots functional, etc. Nice improvements!

**Pull request merged**